unreal_qt_stylesheet/main.py

```diff
@@ -0,0 +1,61 @@
+# import sys
+import os
+
+
+try:
+    from PySide2 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt5 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PySide6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+try:
+    from PyQt6 import QtWidgets, QtCore, QtGui
+except ImportError:
+    pass
+
+# from Qt import QtWidgets, QtCore, QtGui
+# from Qt import _loadUi
+from pathlib import Path
```

Comment:
I assume pathlib is the only Python 2 incompatible code?

Reply:
happy to change that back, or we can point people to the [pathlib](https://pypi.org/project/pathlib/) module for python 2

Reply:
changed it to not use pathlib

Reply:
Thanks, I was just suggesting we remove pathlib and use os.path but you already did that :)
Also, I'm thinking of qt4 compatibility, I've always been using this [Qt.py](https://github.com/mottosso/Qt.py) shim for qt bindings, this only introduces one dependency. Let me know if you think it's worth it

Reply:
I've used the Qt.py dependency a few times myself. 
My gut reaction is that IMO we don't need it, because all we do is import qwidget and qgui.
But my brain tells me I can't answer this without testing in all qt versions.

Qt.py however doesn't yet support pyqt6 / pyside6.
There's also the newer [Qt5.py](https://github.com/mottosso/Qt5.py) but also no pyside6 support.

If we do end up using it, we could always vendor it.

Reply:
made tests, and tested with all qt versions (except qt4 since it's so old I wasnt able to install it, but made tests for it anyone else can run)

we are now good to go without `Qt.py`

Reply:
thank you for testing!